job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1608,6 +1608,78 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+        /**
+          * <p>Configures a Weblogic deployment using the weblogic-deployer-plugin</p>
+          * <p>By default the following values are applied. If an instance of a
+          * closure is provided, the values from the closure will take effect.</p>
+          *
+          * <pre>
+          * {@code
+          * <org.jenkinsci.plugins.deploy.weblogic.WeblogicDeploymentPlugin>
+          *     <mustExitOnFailure>false</mustExitOnFailure>
+          *     <forceStopOnFirstFailure>false</forceStopOnFirstFailure>
+          *     <selectedDeploymentStrategyIds/>
+          *     <isDeployingOnlyWhenUpdates>false</isDeployingOnlyWhenUpdates>
+          *     <deployedProjectsDependencies></deployedProjectsDependencies>
+          *     <tasks>
+          *         <org.jenkinsci.plugins.deploy.weblogic.data.DeploymentTask>
+          *             <id>qGVVG2aOeF</id>
+          *             <taskName>Deploy myApp</taskName>
+          *
+          *             <weblogicEnvironmentTargetedName>environment name</weblogicEnvironmentTargetedName>
+          *             <deploymentName>application name</deploymentName>
+          *             <deploymentTargets>myManagedServer, AdminServer</deploymentTargets>
+          *             <isLibrary>false</isLibrary>
+          *             <builtResourceRegexToDeploy>myApp\.ear</builtResourceRegexToDeploy>
+          *             <baseResourcesGeneratedDirectory></baseResourcesGeneratedDirectory>
+          *
+          *             <jdk>
+          *               <!-- When leaving these tags empty, the default JDK should be used.
+          *                    Otherwise name and home must be set. -->
+          *               <name>JDK 7u51</name>
+          *               <home></home>
+          *               <properties/>
+          *             </jdk>
+          *
+          *             <stageMode>stage</stageMode>
+          *             <commandLine></commandLine>
+          *             <deploymentPlan></deploymentPlan>
+          *         </org.jenkinsci.plugins.deploy.weblogic.data.DeploymentTask>
+          *         <org.jenkinsci.plugins.deploy.weblogic.data.DeploymentTask>
+          *           ...
+          *         </org.jenkinsci.plugins.deploy.weblogic.data.DeploymentTask>
+          *     </tasks>
+          * </org.jenkinsci.plugins.deploy.weblogic.WeblogicDeploymentPlugin>
+          *}
+          * </pre>
+          *
+          * @since 1.39
+          * @see https://wiki.jenkins-ci.org/display/JENKINS/WebLogic+Deployer+Plugin
+          */
+        @RequiresPlugin(id = 'weblogic-deployer-plugin', minimumVersion = '2.9.1')
```

Comment:
The [Weblogic Deployer Plugin wiki page](https://wiki.jenkins-ci.org/display/JENKINS/WebLogic+Deployer+Plugin) claims that newer versions of the pluign are not compatible with 2.x versions. Does your code also work with newer plugin version?